### DISCUSSION TITLE
Légalité du scraping

### DISCUSSION THREAD

**Discussion by AoooR42** [pretrehr/Sports-betting#20] [General]
Bonjour, dans le fichier README vous précisez "Ce projet a pour but d'aider l'utilisateur à dégager de l'argent en se rapprochant au maximum de l'absence de risque. Néanmoins, il est important de préciser que le risque zéro n'existe pas et que les cotes publiées par les bookmakers sont destinées à évoluer au cours du temps. Il est donc de votre ressort de vous assurer que les informations affichées par l'application Sports-betting sont fiables. En tant que créateur, je ne pourrai, en aucun cas, être tenu responsable de toute perte de capital survenue lors de l'utilisation de ce projet.". Mais si l'utilisateur dégage des bénéfices grâce à votre programme, et ce que vous pouvez être accusé d'être responsable de ses gains et être attaqué en justice par les books ? Je vous demande cela car j'ai moi aussi un projet de scraping des books, dans le buts de trouver des surebet et les partager sur un site internet, et je me demande si je peux remettre la faute sur les utilisateurs pour ne pas que mon site soit illégal. Merci pour vos renseignements sur le sujet :)

**Comment by pretrehr** [comment]
Bonjour, 
C'est une question très intéressante. Le fait de récupérer des cotes des différents bookmakers puis de les utiliser comme bon vous semble est tout à fait légal car il s'agit d'informations publiques. (cf. cet article de loi : https://francearchives.fr/fr/article/28204488).
À travers votre projet, vous ne faites qu'afficher ces données publiques. C'est uniquement de la responsabilité des bookmakers s'il existe des surebets sur certains matchs. Quelqu'un qui utilise un surebet est parfaitement dans la légalité. Un bookmaker ne peut donc pas vous reprocher le fait que quelqu'un profite de surebets sur leur site.

Pour plus de tranquillité, rien ne vous empêche d'ajouter un paragraphe dans les "Mentions légales" sur votre site du type : 
"Les cotes recensées sur le site ne sont fournies qu'à titre informatif. Nous ne saurons être tenus responsables par quelque partie en cas d'incident lié à l'utilisation des données présentes sur le site."

N'hésitez pas si vous avez d'autres questions 😃

**Reply by AoooR42** [reply]
> Bonjour,
> C'est une question très intéressante. Le fait de récupérer des cotes des différents bookmakers puis de les utiliser comme bon vous semble est tout à fait légal car il s'agit d'informations publiques. (cf. cet article de loi : https://francearchives.fr/fr/article/28204488).
> À travers votre projet, vous ne faites qu'afficher ces données publiques. C'est uniquement de la responsabilité des bookmakers s'il existe des surebets sur certains matchs. Quelqu'un qui utilise un surebet est parfaitement dans la légalité. Un bookmaker ne peut donc pas vous reprocher le fait que quelqu'un profite de surebets sur leur site.
> 
> Pour plus de tranquillité, rien ne vous empêche d'ajouter un paragraphe dans les "Mentions légales" sur votre site du type :
> "Les cotes recensées sur le site ne sont fournies qu'à titre informatif. Nous ne saurons être tenus responsables par quelque partie en cas d'incident lié à l'utilisation des données présentes sur le site."
> 
> N'hésitez pas si vous avez d'autres questions 😃

Merci pour votre réponse ! ca me rassure énormément car j'ai lu à droite à gauche que la question de la légalité du scraping était ambigüe. Et c'est difficile de trouver des infos à ce sujet sur internet. Je vais essayer de quand même trouver d'autres sources pour être certain, car le risque est énorme. Je veux pas être responsable si les books perdent des dizaines de milliers d'euros ^^. En tout cas merci beaucoup pour votre projet ! J'ai énormément appris de choses et développé mes compétences, autant sur le plan technique avec l'utilisation de sélénium et du multiproccessing que sur le plan organisation avec github. Bonne continuation dans le développement de sports-betting ! :)

**Reply by pretrehr** [reply]
Merci à vous ! Bonne continuation également ;)

**Reply by AoooR42** [reply]
Salut, je reviens vers toi car j'ai contacté les books à propos de la légalité du scraping, et PMU m'a répondu : "Conformément au droit applicable à la propriété littéraire et artistique et à la protection juridique des bases de données, vous ne pouvez, sous peine de poursuites, utiliser que pour vos besoins propres, à l'exclusion de toute utilisation publique, ou exploiter directement ou indirectement à des fins commerciales ou non, toutes les informations que vous pourriez obtenir par la consultation de notre site internet ou autre application." donc à priori, on risquerais des poursuites judiciaires en scrapant les books :/ fait quand même attention, tu risques peut-être gros en proposant publiquement sports-betting. Si tu veux en discuter plus facilement tu peux m'ajouter sur discord : @Rémi#2483 :)
